layers/+misc/multiple-cursors/README.org

```diff
@@ -60,3 +64,44 @@ For easy navigation you also have the following:
 | ~C-t~       | evil-mc-skip-and-goto-next-match   |
 | ~C-M-j~     | evil-mc-make-cursor-move-next-line |
 | ~C-M-k~     | evil-mc-make-cursor-move-prev-line |
+
+** =multiple-cursors=
+The =multiple-cursors= backend provides the following key bindings to
+insert new cursors:
+
+| Key binding | Description                     |
+|-------------+---------------------------------|
+| ~s m m~     | mc/mark-more-like-this-extended |
```

Comment:
The convention is to include `SPC` in key bindings:
`SPC s m m`
etc...

>To keep things readable only mention the prefix SPC when documenting key bindings, you don’t need to mention M-m.

https://github.com/syl20bnr/spacemacs/blob/c7a103a772d808101d7635ec10f292ab9202d9ee/doc/CONVENTIONS.org#writing-documentation

Reply:
Since you've added `SPC` I say leave it, then it at least matches the `c-c++` layer.

Reply:
Done. Since I use Emacs editing style, my brain automatically casts `g r` to `SPC g r` 😄